src/main/java/mekanism/common/block/BlockPlastic.java

```diff
@@ -20,17 +22,14 @@
 
 public class BlockPlastic extends Block
 {
-	public BlockPlastic()
+	public BlockPlastic(String type)
 	{
 		super(Material.wood);
-		setHardness(this == MekanismBlocks.ReinforcedPlasticBlock ? 50F : 5F);
```

Comment:
The value of `this` returned `null` everytime. You can see how that's a problem here.